core/io/image_loader.cpp

```diff
@@ -62,7 +62,7 @@ Error ImageLoader::load_image(String p_file, Ref<Image> p_image, FileAccess *p_c
 
 		if (!loader[i]->recognize(extension))
 			continue;
-		Error err = loader[i]->load_image(p_image, f, p_force_linear);
+		Error err = loader[i]->load_image(p_image, f, p_force_linear, p_scale);
```

Comment:
I agree that it might be the simplest for now but I really don't like that passing a scale for the png import would result in no scaling. so we might at least implement the scaling for all classes so it works as expected. (just with: `resize`)